pkg/pi/pi.go

```diff
@@ -161,3 +164,24 @@ func (p *Pi) openEtcd() *Pi {
 	p.etcd = e
 	return p
 }
+
+func (p *Pi) GetBasicCfg() *config.BasicConfig {
+	p.cfg = config.LoadConf()
+	basicConfig := &config.BasicConfig{
+		PlatformName: p.cfg.Basic.PlatformName,
+		PlatformUrl:  p.cfg.Basic.PlatformUrl,
+	}
+	return basicConfig
+}
+
+func (p *Pi) SetBasicCfg(basicConfig config.BasicConfig) error {
+	if !govalidator.IsURL(basicConfig.PlatformUrl) {
+		return gerr.New(nil, gerr.InvalidArgument, gerr.ErrorIllegalUrlFormat, basicConfig.PlatformUrl)
+	}
+	mutex.Lock()
+	os.Setenv("OPENPITRIX_BASIC_PLATFORM_NAME", basicConfig.PlatformName)
+	os.Setenv("OPENPITRIX_BASIC_PLATFORM_URL", basicConfig.PlatformUrl)
+	config.LoadConf()
```

Comment:
Put global config into etcd, not just system environment. Should not lose when restart container.